react/features/base/react/components/web/InlineDialogFailure.tsx

```diff
@@ -1,13 +1,33 @@
-import React, { Component } from 'react';
-import { WithTranslation } from 'react-i18next';
+/* @flow */
 
-import { translate } from '../../../i18n/functions';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { makeStyles } from 'tss-react/mui';
+
+import { withPixelLineHeight } from '../../../styles/functions.web';
 import Button from '../../../ui/components/web/Button';
 
+const useStyles = makeStyles()(theme => {
+    return {
+        dialog: {
+            backgroundColor: theme.palette.ui01,
+            border: `1px solid ${theme.palette.ui04}`,
+            borderRadius: `${Number(theme.shape.borderRadius)}px`,
+            boxShadow: '0px 1px 2px rgba(41, 41, 41, 0.25)',
+            color: theme.palette.text01,
+            ...withPixelLineHeight(theme.typography.bodyShortRegular),
+            padding: `${theme.spacing(3)} 10`,
+            '& button': {
+                margin: '16px auto 0 auto'
+            }
+        }
+    };
+});
+
 /**
  * The type of the React {@code Component} props of {@link InlineDialogFailure}.
  */
-interface IProps extends WithTranslation {
+type Props = {
```

Comment:
Please leave it as interface. We generally try to use interfaces everywhere and types only where we can't use interfaces